spec/requests/create_subscriber_list_spec.rb

```diff
@@ -134,14 +136,21 @@
         expect(response.status).to eq(201)
       end
     end
+  end
+
+  context "when creating a subscriber list with 'email' and 'government' document supertypes" do
+    it "returns a 201" do
+      create_subscriber_list(
+        email_document_supertype: "publications",
+        government_document_supertype: "health",
```

Comment:
Could we use an actual gov supertype here eg `news-stories`, to avoid confusion?